packages/image/src/__test__/load.test.ts

```diff
@@ -0,0 +1,49 @@
+import { afterAll, beforeAll, describe, expect, it } from "vitest";
+import { type ImageSource, load } from "../load";
```

Comment:
```suggestion
import { type ImageSource, load } from "..";
```

To test library importing from index is better I think